sample/App.fs

```diff
@@ -49,10 +63,58 @@ let view model dispatch =
 // |> Program.run
 
 [<LitElement("sample-app")>]
-let App() =
-    Hook.useHmr(hmr)
-    let _ = LitElement.init(fun config ->
-        config.useShadowDom <- false
-    )
-    let model, dispatch = Hook.useElmish(init, update)
-    view model dispatch
\ No newline at end of file
+let App () =
+    Hook.useHmr (hmr)
+    let _ = LitElement.init (fun config -> config.useShadowDom <- false)
+    let model, dispatch = Hook.useElmish (init, update)
+    view model dispatch
+
+open Experimental
+
+[<LitElementExperimental(true)>]
+let Sample () =
+    let name = Controllers.GetProperty("name", "Peter")
+
+    let state: StateController<int> =
+        Controllers.GetController<StateController<int>>(10)
+
+    let twoArgs = Controllers.GetController<TwoArgsCtrl>(10, 20)
```

Comment:
I was thinking on something like this to prevent having the LitElement generic initialization, we still do magic, but more _javascrip'ty_ magic (or that's what I want to believe) in the end these are just properties that are in the class that is being used behind this function to use lit's mechanisms without diving too much into directive territory

Side note perhaps _**Context**_ would be a better name rather than _Controllers_